superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -432,6 +432,29 @@ export const controls = {
     'to find in the [country] column'),
   },
 
+  freq: {
+    type: 'SelectControl',
+    label: t('Frequency'),
+    default: 'W-MON',
+    freeForm: true,
+    clearable: false,
+    choices: [
+      ['AS', '[AS] Year'],
```

Comment:
what does `AS` mean here?
EDIT: I see it's pandas jargon. Can you just encode `AS` as the value and remove `AS` just using `year` in the label? novice users are not going to understand that and even though it says `year` currently they might think it's doing something else confusing and be unsure of their selection because it says `AS`

Reply:
I get that it's confusing, and was trying to be transparent about the pandas freq mapping. Especially given that this is a freeform dropdown allowing them to use whatever valid pandas freq. Maybe moving to `Year (freq=AS)` and following that pattern?

Reply:
I agree transparency is valuable. I think that pattern is a good middle ground!